index.html

```diff
@@ -8,29 +8,157 @@
     <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-   <section class="body">
-    <header>
-      <a href="#" class="welcome">Hi there</a> 
-      <nav>
-        <a href="#"><img src="images/Union.png" class="navigation_bar" alt="Navigation bar"/></a> 
-      </nav>
-    </header>
-       <section class="dashboard">
-        <div class="backgr">
-            <img src="images/Ellipse 18.png" alt="Ellipse 18">
+  <div class="body">
+    <section class="body">
+      <header>
+        <a href="#" class="welcome">Hi there</a> 
+        <nav>
+          <a href="#"><img src="images/Union.png" class="navigation_bar" alt="Navigation bar"/></a> 
+        </nav>
+      </header>
+         <section class="dashboard">
+          <h1 class="name">Abednego Kibitok</h1>
+          <h2 class="title">Software Developer</h2>
+          <p class="pa">
+              I can help you build a product ,<br>
+               feature or website Look through<br>
+                some of my work and experience! If<br>
+                 you like what you see and have a<br>
+                  project you need coded,  don’t<br>
+                   hestiate to contact me.
+          </p>
+          <img src="images/Vector 4.png" class="line" alt="Horizontal line">
+         </section>
+     </section>
+    <section class="skill">
+      <section class="languageframeworks">
+        <ul class="language">
+          <li id="language">Language</li>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+        <ul class="language">
+          <li id="language">Frameworks</li>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+        <img src="images/Ellipse 19.png" class="ellipse" alt="Ellipse">
+      </section>
+      <section class="skills">
+        <ul class="language">
+          <li id="language">Skills</li>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlap</li>
+          <li>Tarminal</li>
+        </ul>
+      </section>
+    </section>
+    <img src="images/Vector 4.png" class="line" alt="Horizontal line">
+    <ul class="socialmedia">
+      <li><a href="#"><img src="images/Vector.png" alt="Github_icon"></a></li>
+      <li><a href="#"><img src="images/Group.png" alt="Linkedin_icon"></a></li>
+      <li><a href="#"><img src="images/VectorX.png" alt="X_icon"></a></li>
+      <li><a href="#"><img src="images/Teiitter.png" alt="Twitter_icon"></a></li>
+      <li><a href="#"><img src="images/Vectorm.png" alt="M_icon"></a></li>
+    </ul>
+    <div class="scroll">
+      <img src="images/Group 55.png" class="arrow" alt="Arrow">
+      <img src="images/Work.png" class="work" alt="work">
+    </div>
+    <div class="text">MY RECENT WORK</div>
```

Comment:
- [ ] Kindly make sure to wrap the `MY RECENT WORK` title into the heading tag `hx`. :wink: